doc/development-workflow.md

```diff
@@ -0,0 +1,22 @@
+# Development workflow
+
+## PR author
+
+1. Open a PR with the changes you want to merge.
+
+2. Add the "Waiting on factcheck" label if the changes require feedback from someone other than developers.
+
+3. Move the associated Trello card from "In progress" to "Code review".
```

Comment:
@pmanrubia That's more confusing.

I'd do the following:

1. Open a PR with the changes.
2. Add a "Ready for Code Review" label when it's ready for review.
3. The code reviewer should then change the label to "Passed code review" when they and the developer have agreed that the PR is good to go.
4. If the PR needs to be looked at by someone other than a dev, the developer should then:
   i) Set the "Waiting for factcheck" label.
   ii) Remove the "Waiting for factcheck" label when the change is signed off from factcheck.
5. If the PR doesn't need a factcheck, or has successfully come back from a factcheck, the developer should then merge the PR.

I would still ditch the LGTM label entirely. It's too confusing to keep around. That label makes you think a PR is ready to be deployed, when that is not necessarily true any longer.